bluesky/callbacks/mpl_plotting.py

```diff
@@ -58,15 +58,16 @@ class Teleporter(QtCore.QObject):
 
 
 class QtAwareCallback(CallbackBase):
-    def __init__(self, *args, use_teleporter=None, **kwargs):
+    def __init__(self, *args, **kwargs):
+        use_teleporter = kwargs.get('use_teleporter', None)
         if use_teleporter is None:
             import matplotlib
             use_teleporter = 'qt' in matplotlib.get_backend().lower()
         if use_teleporter:
             self.__teleporter = _get_teleporter()
         else:
             self.__teleporter = None
-        super().__init__(*args, **kwargs)
+        super().__init__()
```

Comment:
Any stray args/kwargs should be passed on to the super, not dropped on the floor here.